packages/js-example-app/src/index.js

```diff
@@ -110,10 +113,46 @@ blrTextArea.addEventListener('blrBlur', () => {
   addLog('blr-textarea blurred');
 });
 
-blrTextArea.addEventListener('blrChange', () => {
-  addLog('blr-textarea changed');
+blrTextArea.addEventListener('blrTextValueChange', (e) => {
+  addLog('blr-textarea changed: ' + e.detail.inputValue);
 });
 
 blrTextArea.addEventListener('blrSelect', () => {
   addLog('blr-textarea selected');
 });
+
+blrRadioInput.addEventListener('blrFocus', () => {
+  addLog('blr-radio focused');
+});
+
+blrRadioInput.addEventListener('blrBlur', () => {
+  addLog('blr-radio blurred');
+});
+
+blrRadioInput.addEventListener('blrSelectedValueChange', (e) => {
+  addLog('blr-radio changed: ' + e.detail.selectedValue);
+});
+
+blrToggleSwitch.addEventListener('blrFocus', () => {
+  addLog('blr-label-toggleswitch focused');
+});
+
+blrToggleSwitch.addEventListener('blrBlur', () => {
+  addLog('blr-label-toggleswitch blurred');
```

Comment:
Can't it be just blr-toggleswitch blurred for the consistency of the naming with others. Same for line 145.